app/services/json_to_events.rb

```diff
@@ -19,7 +19,7 @@ def call
       campus = Campus.where(name: event_hash["campus"]).first_or_create!
 
       # Create an event
-      Event.create!(
+      event = Event.create!(
```

Comment:
Do we still need to set this variable?

Reply:
No I think that was just to output data since we didn't know how to view the db items